test/grammar/builtins/file/rmdir/stdout.golden

```diff
@@ -0,0 +1 @@
+
```

Comment:
Remove empty stdout file.

Reply:
I have removed it!

Reply:
Remove the test/grammar/builtins/file/rmdir/stdout.golden ...